app/goal_routes.py

```diff
@@ -0,0 +1,107 @@
+from os import abort
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+from flask import Blueprint, jsonify, abort, make_response, request
+from datetime import datetime
+import requests  
+import os
+
+
+goals_bp = Blueprint("goal", __name__, url_prefix="/goals")
+
+
+@goals_bp.route("", methods=["POST"])
```

Comment:
👍 I like this organization here! Good guard clause, organized, helper functions. Looks great!